src/content/release-notes.mdx

```diff
@@ -5,6 +5,35 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.18.0
+
+27 January 2024
+
+This version is compatible with Kubernetes versions 1.25 to 1.28.
+
+### New Features
```

Comment:
@codyjlandstrom I would call out the Argo CD installation guide. An alternative is to mentioning it in the release thena message, but I think it's important to let people know about this guide

Reply:
Good call. Will add this now